src/http_request.rs

```diff
@@ -7,8 +7,8 @@ use std::{
 use tracing::info;
 
 pub struct HttpRequest {
-    path: Option<String>,
-    method: Option<MethodType>,
+    path: String,
+    method: MethodType,
```

Comment:
Could add here the `params` fields to the struct with the `hashmap<String, String>` type? This field will store information about the request arguments parsed from the incoming request (I mean the arguments from the URL like `?sort_by=type&ordering=id,timestamp`)

Reply:
Yes I think it can be added, but I have no usecase actually, just want to handle a simple GET request on / path.

Reply:
You will get usecases by time. :) One at a time. Next step is query string parsing :)